MACE_Frontend/src/components/map/index.tsx

```diff
@@ -65,29 +73,78 @@ export default class MapView extends React.Component<Props, State> {
     if(this.state.contextMenuVisible != nextState.contextMenuVisible) {
         return true;
     }
+    if(this.state.popupVisible != nextState.popupVisible) {
+      return true;
+    }
     return false;
   }
 
+  togglePopup = () => {
+    this.setState({selectedAircraft: "ALL"});
+    this.setState({ popupVisible: !this.state.popupVisible });
+  }
+
+  onAircraftSelected = (e) => {
+    this.setState({selectedAircraft: e.target.value });
+  }
+  createDropdownList = () => {
+    let options = [];
+    let hide = false;
+    options.push(<option key={0} value="ALL">All Aircraft</option>);
+    this.props.context.aircrafts.forEach((a) => {
+      if (["STABILIZE",""].indexOf(a.mode) != -1){
```

Comment:
Why are you checking for `STABILIZE`? I think the dropdown should just display all aircraft.